include/deal.II/lac/trilinos_tpetra_sparse_matrix.templates.h

```diff
@@ -430,7 +430,8 @@ namespace LinearAlgebra
 
     template <typename Number, typename MemorySpace>
     template <typename SparsityPatternType>
-    inline void
```

Comment:
I'm removing the `inline` here because we're in a `.template.h` file, and because inlining a function that (forwards to a function that) does a lot of work is not going to help much anyway.